OpenSim/Simulation/Model/GeometryPath.cpp

```diff
@@ -492,8 +492,10 @@ addPathPoint(const SimTK::State& s, int aIndex, const PhysicalFrame& frame)
 {
     PathPoint* newPoint = new PathPoint();
     newPoint->setParentFrame(frame);
-    Vec3 location = newPoint->getLocation(s);
+    Vec3 location = newPoint->getLocation(s); // 0 by construction
```

Comment:
I don't understand this comment. Where would you expect the location to be if none was set?

Reply:
You do not even need to as newPoint for its location since you know you are going to have to set it correctly.  It would be more readable if this was just `Vec3 newLocation;`